lib/cocoapods/project.rb

```diff
@@ -183,14 +184,7 @@ def add_file_reference(absolute_path, group, reflect_file_system_structure = fal
         raise ArgumentError, "Paths must be absolute #{absolute_path}"
```

Comment:
True ... Except the stack trace being a little more concise when the error is thrown, there is no reason to keep it there.